hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFormatReader.java

```diff
@@ -52,9 +53,9 @@ public class HoodieLogFormatReader implements HoodieLogFormat.Reader {
 
   private static final Logger LOG = LogManager.getLogger(HoodieLogFormatReader.class);
 
-  HoodieLogFormatReader(FileSystem fs, List<HoodieLogFile> logFiles, Schema readerSchema, boolean readBlocksLazily,
```

Comment:
we can remove internalSchema directly.
internralSchema is only used to indicate current hudi table has done schema evolution. and we should use writeSchema to read logblock.

if schemaEvolution is disabled, let's pass readerSchema as null